### PR TITLE
RE2022-245: save GTDB metadata to import_files dir

### PR DESCRIPTION
**Pull request by Tianhao-Gu**

### CODE REVIEW

File: src/loaders/gtdb/gtdb_genome_attribs_loader.py [hunk 1]
```diff
@@ -151,10 +154,14 @@ def main():
...
+    root_dir = getattr(args, loader_common_names.ROOT_DIR_ARG_NAME)
+
+    attri_output = f'{kbase_collection}_{load_version}_{GTDB_GENOME_ATTR_FILE}'
+    loader_helper.create_import_files(root_dir, env, attri_output, docs)
 
-    # TODO save columnar_attri_meta to import_files/NONE folder
-    with args.output as genome_attribs_json:
-        loader_helper.convert_to_json(docs, genome_attribs_json)
+    meta_output = f'{kbase_collection}_{load_version}_{GTDB_GENOME_ATTR_META_FILE}'
+    loader_helper.create_import_files(root_dir, env, meta_output, [meta_doc])
```
Comment (MrCreosote) [inline src/loaders/gtdb/gtdb_genome_attribs_loader.py:164]: Can we format these the same way as the other import files and include the target Arango collection in the name?

**Comment by MrCreosote** [comment]
Tests are consistently failing

File: src/loaders/gtdb/gtdb_genome_attribs_loader.py (hunk 1 above)
Comment (Tianhao-Gu) [reply src/loaders/gtdb/gtdb_genome_attribs_loader.py]: 👍

**Comment by Tianhao-Gu** [comment]
> Tests are consistently failing

Fixed tests.

**Review by MrCreosote — APPROVED** [review]

**Pull request merged**